xapi-tapctl.opam

```diff
@@ -1,24 +1,18 @@
 opam-version: "1.2"
-maintainer: "jonathan.ludlam@eu.citrix.com"
+name: "xapi-tapctl"
+maintainer: "jonathan.ludlam@citrix.com"
 authors: "xen-api@lists.xen.org"
 homepage: "https://xapi-project.github.io/"
 bug-reports: "https://github.com/xapi-project/tapctl/issues"
 dev-repo: "git://github.com/xapi-project/tapctl.git"
 tags: [ "org:xapi-project" ]
-build: [
-  ["oasis" "setup"]
-  [make]
-]
-install: [
-  [make "install" "BINDIR=%{bin}%"]
-]
-remove: [
-  ["ocamlfind" "remove" "tapctl"]
-]
+build: [[ "jbuilder" "build" "-p" name "-j" jobs ]]
 depends: [
-  "ocamlfind" {build}
-  "oasis" {build}
-  "xapi-stdext" {>= "2.0.0"}
+  "jbuilder" {build}
+  "xapi-stdext-unix"
+  "xapi-stdext-std"
+  "xapi-stdext-threads"
+  "xapi-stdext-monadic"
   "xapi-forkexecd"
```

Comment:
I think `"forkexec"` is enough for this package, but whoever used it will need the daemons as well, so `xapi-forkexecd` could be fine as well

Reply:
Oh, we are dropping the dependency... than let's not waste time on it :)